src/components/periodic-table-page/element-info/ElementInfo.tsx

```diff
@@ -11,11 +11,11 @@ interface IElementInfoDataEntryProps {
   unit?: string;
 }
 
-const ElementInfoDataEntry: React.StatelessComponent<IElementInfoDataEntryProps> = ({
+const ElementInfoDataEntry = ({
   name,
   value,
   unit,
-}) => {
+}: IElementInfoDataEntryProps) => {
```

Comment:
It's not required if you want to move on, but we've also been removing the `I` prefix on types on files we're touching.

Reply:
Done :)

Reply:
Have you pushed it? It still says `IElementInfoDataEntryProps` and not `ElementInfoDataEntryProps` for me in this review